backbone/src/main/java/org/researchstack/backbone/utils/FormatHelper.java

```diff
@@ -8,6 +11,12 @@
 
 public class FormatHelper {
 
+    private static final String COUNTRY_US      = "US";
+    private static final String COUNTRY_LIBERIA = "LR";
```

Comment:
Believe it or not...

https://www.google.com/search?client=safari&rls=en&q=which+countries+use+the+imperial+unit+system&ie=UTF-8&oe=UTF-8